modules/thorvg/image_loader_thorvg.h

```diff
@@ -28,46 +28,28 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
-#ifndef IMAGE_LOADER_SVG_H
-#define IMAGE_LOADER_SVG_H
+#ifndef RESOURCE_IMPORTER_THORVG
+#define RESOURCE_IMPORTER_THORVG
 
+#include "core/error/error_macros.h"
 #include "core/io/image_loader.h"
 #include "core/string/ustring.h"
 
-/**
-	@author Daniel Ramirez <djrmuv@gmail.com>
-*/
-
-// Forward declare and include thirdparty headers in .cpp.
-struct NSVGrasterizer;
-struct NSVGimage;
-
-class SVGRasterizer {
-	NSVGrasterizer *rasterizer;
+#include "core/templates/local_vector.h"
+#include "thirdparty/thorvg/inc/thorvg.h"
+#include <stdint.h>
+#include <iostream>
+#include <memory>
+#include <sstream>
```

Comment:
None of these are used? This is a very simple header, I wouldn't expect much more than `"core/io/image_loader.h"`.